test/test_unistd.rs

```diff
@@ -64,7 +64,12 @@ fn test_wait() {
 
 #[test]
 fn test_mkstemp() {
-    let result = mkstemp("/tmp/nix_tempfile.XXXXXX");
+    #[cfg(target_os = "android")]
```

Comment:
Shouldn't this use [`std::env::temp_dir`](https://doc.rust-lang.org/std/env/fn.temp_dir.html) instead?